ansible/roles/ceph-release/tasks/main.yml

```diff
@@ -1,91 +1,9 @@
-  git:
-    repo: https://github.com/ceph/ceph
-    remote: upstream
-    dest: ceph
-    accept_hostkey: yes
-  when: ceph_repo.stat.exists is defined and ceph_repo.stat.exists == false
-
-- name: add origin
-  command: git remote add origin git@github.com:ceph/ceph-releases.git chdir=ceph
-  # because if the repo exists then it probably has the origin already in there
-  ignore_errors: yes
-
-- name: reset --hard to upstream
-  command: git reset --hard origin/{{ branch }} chdir=ceph
-  ignore_errors: yes
-  when: clean
-
-- name: force git checkout {{ branch }} branch
-  command: git checkout -f {{ branch }} chdir=ceph
-
-- name: fetch upstream
-  command: git fetch upstream -v chdir=ceph
-
-- name: git submodule update
-  command: git submodule update --init chdir=ceph
-
-- name: check if configure.ac exists (pre-kraken)
-  stat: path=ceph/configure.ac
-  register: configure_ac
-
-- name: replace the version in configure.ac (pre-kraken)
-  lineinfile: dest=ceph/configure.ac
-              regexp='^AC_INIT\(\[ceph\],'
-              line='AC_INIT([ceph], [{{ version }}], [ceph-devel@vger.kernel.org])'
-  when: configure_ac.stat.exists
-
-- name: check if CMakeLists.txt exists
-  stat: path=ceph/CMakeLists.txt
-  register: cmake_lists
-
-- name: replace the version in CMakeLists.txt
-  lineinfile: dest=ceph/CMakeLists.txt
-              regexp='^  VERSION \d+\.\d+\.\d+$'
-              line='  VERSION {{ version }}'
-  when: cmake_lists.stat.exists
-
-- include: release/candidate.yml
-  when: "release in ['candidate', 'rc']"
-
-- include: release/development.yml
-  when: "release == 'development'"
-
-- include: release/stable.yml
-  when: "release == 'stable'"
-
-- name: commit the version changes
-  command: git commit -a -m "{{ version }}" chdir=ceph
-
-- name: clear the previous local tag
-  command: git tag -d v{{ version }} chdir=ceph
-  ignore_errors: yes
-  when: clean
-
-- name: clear the previous remote tag
-  command: git push origin :v{{ version }} chdir=ceph
-  ignore_errors: yes
-  when: clean
-
-  # from script: /srv/ceph-build/tag_release.sh
-- name: tag the version
-  # we used to sign releases like:
-  # GNUPGHOME=~/build/gnupg.ceph-release
-  # git tag -s "v{{ version }}"  -u 17ED316D -m "v{{ version }}" chdir=ceph
-  command: git tag "v{{ version }}" -m "v{{ version }}" chdir=ceph
-
-  # 'origin' in this case is ceph-releases, since this is our fork/copy of the
-  # official ceph repo it is ok to forcefully push "losing" changes. This may
-  # come up if we need to re-do a release which means doing a fresh commit to
-  # the debian changelog and rpm files and tagging.
-- name: force push changes to ceph-releases git repo
-  command: git push -f origin {{ branch }} chdir=ceph
-
-- name: push the newly created tag
-  command: git push origin v{{ version }} chdir=ceph
+- import_tasks: create.yml
+  when: "stage == 'create'"
+
+- import_tasks: push.yml
+  when:
+    - "stage == 'push'"
+    - "release != 'SECURITY'"
```

Comment:
It sort of is normalized. It will only and always be passed in all caps: https://github.com/ceph/ceph-build/pull/2000/files#diff-ac9c3d44df3ef56e6433aecfc057f89efe44a9cd6335f21caf9c7b03d33fe47fR72